reana_server/openapi_connections/reana_workflow_controller.json

```diff
@@ -352,6 +454,223 @@
         "summary": "Adds a file to the workflow workspace."
       }
     },
+    "/api/workflows/{workflow_id}/workspace/inputs": {
```

Comment:
This should be added when addressing https://github.com/reanahub/reana-server/issues/20, please remove :)